src/main/java/com/shop/model/Image.java

```diff
@@ -14,12 +14,9 @@
     @Column(name = "image_id")
     private long id;
 
-    @Lob
-    @NotBlank
     private String url;
 
```

Comment:
It depends on host which we will be using, but some of them have links with more than 255 characters, so clob is not the worst idea.